main.go

```diff
@@ -262,7 +276,28 @@ func stdHandler(ctx context.Context, okCh chan<- int, errCh chan<- error) io.Wri
 					okCh <- int(attr["port"].(float64))
 				}
 
-				log.Info(ctx, fmt.Sprintf("[mongod] %s", message), logMessage)
+				switch minLogLvl {
+				case Dbg:
+					log.Info(ctx, fmt.Sprintf("[mongod] %s", text))
```

Comment:
Hi, thanks for the patch (and the nudge).

```suggestion
					log.Info(ctx, fmt.Sprintf("[mongod] %s", message), logMessage)
```

I wonder why you're only logging `text` here (and in lines below).
I like the idea of dealing better with the levels, but the `log.Info` that you've replaced
had `message` and `logMessage` (i.e. not 'just' `text`), so why this change?

Admittedly, this looks repetitive, and wonder if we should `delete(logMessage, "msg")` after extracting that to `message` in the above?

Caveat: I'm not familiar with this lib

Reply:
Great spot! And thanks for getting back to me. I see what you're saying. Looks like I pulled in the wrong variable. 

I'll double check, update and push

Reply:
I've updated logs to use `message` not `text`, and added a comment to highlight why info is used when no severity can be extracted.

Interestingly I noticed the ONS logger has no DEBUG level, hence my logging DEBUG messages to INFO. I _could_ include an indication that a DEBUG message is DEBUG, however that'd mess up any formatting/alignment so have chosen to leave it.

Reply:
Thanks for the PR - finally got it in, today.

Worth noting that I reworked the logic because your code would `log.Info` when the _desired log level_ was `Inf`, even if (despite) the message itself (`severity`) was an error (`E`).

Reply:
Another good spot. Thanks @gedge !